enterprise/frontend/src/embedding-sdk/components/public/InteractiveDashboard/InteractiveDashboard.tsx

```diff
@@ -23,10 +23,7 @@ import type { PublicOrEmbeddedDashboardEventHandlersProps } from "metabase/publi
 import { InteractiveDashboardProvider } from "./context";
 
 export type InteractiveDashboardProps = {
-  drillThroughQuestionHeight?: number;
   plugins?: SdkPluginsConfig;
-  className?: string;
-  style?: CSSProperties;
```

Comment:
Moved this to `SdkDashboardDisplayProps` as it's used in both static and interactive